test/modules/TestResultCoordinator/Reports/DirectMethod/DirectMethodTestOperationResultComparer.cs

```diff
@@ -33,7 +33,8 @@ public bool Matches(TestOperationResult value1, TestOperationResult value2)
             DirectMethodTestResult dmtr1 = JsonConvert.DeserializeObject<DirectMethodTestResult>(value1.Result);
             DirectMethodTestResult dmtr2 = JsonConvert.DeserializeObject<DirectMethodTestResult>(value2.Result);
 
-            return string.Equals(dmtr1.SequenceNumber, dmtr2.SequenceNumber, StringComparison.OrdinalIgnoreCase);
+            return string.Equals(dmtr1.SequenceNumber, dmtr2.SequenceNumber, StringComparison.OrdinalIgnoreCase)
```

Comment:
This is not an operation result comparer; I think it should rename as sth for matching message identity, e.g. DirectMethodTestOperationResultIdentityComparer.

Reply:
Removed whole class